README.md

```diff
@@ -49,18 +49,12 @@
     </h4>
 </div>
 
-| Groups are an important concept when we speak about privacy and zero knowledge technologies. They can be thought of as anonymity sets, and are a way to establish necessary trust between a set of participants. The goal of this project is to provide a comprehensive infrastructure to allow anyone to create and manage their own groups. |
-| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Bandada is a public good plug-and-play infrastructure that empowers anyone to create and manage privacy-preserving groups of anonymous individuals, without necessitating prior expertise. |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 
-Bandada consists of a back-end to store the groups and provide the [**API**](/apps/api), two front-ends: the [**dashboard**](/apps/dashboard) to manage groups and members and a [**demo**](/apps/client) application to allow end-users to join the groups, and the [**contracts**](/apps/contracts) Additionally, it also provides a set of JavaScript libraries to support developers.
+Bandada comprises a versatile back-end, two user-friendly front-ends (the [**dashboard**](/apps/dashboard) to manage groups and members and a [**demo**](/apps/client) application to allow end-users to join the groups), Ethereum [**smart contracts**](/apps/contracts) for proof verification, and a collection of [**JavaScript libraries**](/libs/) for seamless integration. From the Bandada dashboard, you can effortlessly create two types of groups: **manual** and **credential** groups. In manual groups, you can add members directly or generate invite links while, in credential groups, members must demonstrate their credentials for access. Bandada provides developers with JavaScript libraries, including [`@bandada/api-sdk`](/libs/api-sdk/) to make it easier to work with the APIs. Furthermore, it provides preconfigured credential validators and allows for additional functionality through the [`@bandada/credentials`](/libs/credentials/) library. This feature allows for manual or automated management of both off-chain and on-chain groups by specifying eligibility criteria. It can be used for a variety of applications, such as organizing private organizational members, grouping contributors of a particular GitHub repository, or uniting holders of a specific NFT, among others.
```

Comment:
`... to make it easier to work with the APIs` -> `... to make it easier to work with the API`. 

Since it is just one API, it would be better to put the singular version. What do you think?

Reply:
Thank you Vivi for the review :sparkle:

Hmm, the SDK has many APIs - so the 's' stands for the set of APIs inside the SDK itself. Does it makes sense?

Reply:
Yes, I get what you mean. What I was saying is that the SDK makes it easier to work with the Bandada API.

Reply:
Ahh, got it! Sorry. Since this PR has been merged, feel free to open an issue to track or just update through `main` (it's quick)!

Reply:
No worries, feel free to do it too 👍